doc/admin-guide/files/records.config.en.rst

```diff
@@ -317,6 +317,11 @@ Thread Variables
    The number of threads |TS| will create if `proxy.config.exec_thread.autoconfig`
    is set to ``0``, otherwise this option is ignored.
 
+.. ts:cv:: CONFIG proxy.config.exec_thread.listen INT 0
+
+   If enabled (``1``) all the exex_threads listen for incoming connections. `proxy.config.accept_threads`
+   should be disabled to enable this variable.
+
```

Comment:
We will have:

| proxy.config.accept_threads | proxy.config.exec_thread.listen | comments |
|:-----------------------------:|:---------------------------------:| ------------ |
| 0 | 0 | accepts will be done in each of the worker threads and share one listen fd with EPOLLEXCLUSIVE if possible. |
| 0 | 1 | accepts will be done in each of the worker threads and dedicated listen fd will be opened with SO_REUSEPORT. |
| 1 | 0 | accepts will be done in dedicated thread and do blocking accept with one listen fd. |

Reply:
This looks good but where do I write this?

Reply:
updated records.config with the above information.